ide-launcher-advanced@adrianopaonessa/files/ide-launcher-advanced@adrianopaonessa/launcher.sh

```diff
@@ -58,23 +65,39 @@ check_more_than_one() {
             if [ "$IDE" -eq "$i" ] ; then
                 echo; echo "Launching ${installedIDEs[$i]}...";
                 
-                if [ "${installedIDEs[$i]}" == "Visual Studio Code" ] ; then code "${CURRENT_DIR}";
-                elif [ "${installedIDEs[$i]}" == "Visual Studio Code Insiders" ] ; then code-insiders "${CURRENT_DIR}";
-                elif [ "${installedIDEs[$i]}" == "VSCodium" ] ; then codium "${CURRENT_DIR}";
-                elif [ "${installedIDEs[$i]}" == "VSCodium Insiders" ] ; then codium-insiders "${CURRENT_DIR}";
+                if [ "${installedIDEs[$i]}" == "Visual Studio Code" ] ; then code "${CURRENT_DIR}"; exit 0;
+                elif [ "${installedIDEs[$i]}" == "Visual Studio Code Insiders" ] ; then code-insiders "${CURRENT_DIR}"; exit 0;
+                elif [ "${installedIDEs[$i]}" == "VSCodium" ] ; then codium "${CURRENT_DIR}"; exit 0;
+                elif [ "${installedIDEs[$i]}" == "VSCodium Insiders" ] ; then codium-insiders "${CURRENT_DIR}"; exit 0;
+                elif [ "${installedIDEs[$i]}" == "Sublime Text" ] ; then subl "${CURRENT_DIR}"; sublime_launched;
```

Comment:
@adrianopaonessa sup.
You can try using `nohup subl "${CURRENT_DIR}" &` to detach the sublime process from the terminal upon launching, making it unnecessary to keep the terminal window open.

Reply:
Hi @anaximeno !
Thank you for your suggestion ^^
I will give it a look when i open my PC.

Reply:
I'm kind of more curious, why you're trying to combine multiple editors into a single action, and prompting which editor to use from a terminal.

In general, when a terminal opens up at any time because of opening a text editor, but then either is used as a prompt, or immediately vanishes (like I've seen with this), then I question why.

The way I started implementing this in an alternative way where I can specifically choose what I want is making an action per IDE editor that supports opening a directory, and instead of giving a prompt, I literally get a list instead, as seen here:

![Nemo Multi-IDE](https://github.com/user-attachments/assets/805f2f51-1e2b-4957-98c6-299dff65ea6b)

Reply:
Which, BTW, I'm recommending an option too, in the examine.sh, when "$WORKING_DIR" == "$HOME" included in the list with "/", to avoid listing it. For people with large $HOME directories, which most tend to become, text editors, like Sublime Text, try to index every single file on load, which Sublime Text will literally OOM out doing this, and re-open again doing the same.